packages/react/package.json

```diff
@@ -2,18 +2,27 @@
   "private": true,
   "name": "@aws-amplify/ui-react",
   "version": "0.0.1",
-  "main": "dist/index.js",
+  "type": "module",
+  "main": "dist/index.cjs",
+  "module": "dist/index.js",
+  "exports": {
+    ".": "./dist/index.js",
+    "./styles.css": "./dist/index.css"
```

Comment:
`import "@aws-amplify/ui-react/styles.css"` correctly resolves to `dist/index.css`.

Neat, right?

Reply:
![neat](https://media.giphy.com/media/h1zJMhT5XOT927e0aw/giphy.gif?cid=ecf05e47aquw2prmqrs06jwt407ozgxykzi9k8pv6h8pqgqj&rid=giphy.gif&ct=g)